sources/lib/datalab/gcp/datalab/_utils.py

```diff
@@ -207,6 +207,8 @@ def parse_config(config, env):
   """ Parse a config from a magic cell body. This could be JSON or YAML. We turn it into
```

Comment:
nit: This could be JSON or YAML -> This could be JSON, YAML, or inline sql query.